In [1]:
name="Smith"
hourly_wage=19.5
hours=20

weekly_pay= hourly_wage*hours

print(f"{name} earns {weekly_pay} per week")

Smith earns 390.0 per week


In [2]:
temps = [61, 65, 72, 58, 70, 74, 68]
greater = 0
for s in temps:
    if s > 65:
        greater = greater + 1
print(greater)

4


In [3]:
def to_celsius(f):
    return (f-32)*5/9
round(to_celsius(temps[0]),1)

16.1

In [4]:
movie = {"title": "Up", "year": 2009, "rating": 8.3}
for key, value in movie.items():
    print(key, ":", value)

title : Up
year : 2009
rating : 8.3


In [5]:
import pandas as pd
df = pd.read_csv("chickwts.csv")
df.head()

,Unnamed: 0,weight,feed
0,1,179,horsebean
1,2,160,horsebean
2,3,136,horsebean
3,4,227,horsebean
4,5,217,horsebean


In [6]:
df.describe()

,Unnamed: 0,weight
count,71.000000,71.000000
mean,36.000000,261.309859
std,20.639767,78.073700
min,1.000000,108.000000
25%,18.500000,204.500000
50%,36.000000,258.000000
75%,53.500000,323.500000
max,71.000000,423.000000


describe() tells us the information of all the weights of all chickens, giving us the minimum and maximum, mean, standard deviation, and the total count of the chickens. Of the 71 total chickens, the mean weight is approximately 261.31, the minimum is 108, and the maximum is 423. The thing that surprised me was the 25%, 50%, and 75% since the describe() not only did calculations, but ordered the weights from least to greatest and selected the ones in those positions exactly.